configure.py

```diff
@@ -1540,8 +1540,11 @@ def write_config(data, name):
     os.mkdir(icu_tmp_path)
     icu_data_path = os.path.join(icu_tmp_path, icu_data_file_l)
     with open(icu_data_path, 'wb') as outf:
-        with bz2.BZ2File(compressed_data, 'rb') as inf:
+        inf = bz2.BZ2File(compressed_data, 'rb')
+        try:
             shutil.copyfileobj(inf, outf)
+        finally:
+            inf.close()
```

Comment:
Can you use two spaces of indent rather than four? I realize you're just following what the previous code did but that code was using the wrong indentation. :-)